src/Material/Button.elm

```diff
@@ -145,6 +232,26 @@ ripple =
     (\options -> { options | ripple = True })
 
 
+{-| Add link-attributes that turn the button from `button` into a `a`-element.
+This allows for links that look and feel like buttons but perform link actions.
+
```

Comment:
Can you still add an `onClick` handler? Maybe the docs should state whether you can and what will happen. Someone will try it at some point (e.g., if the link only does internal navigation in their SPA and they want something else to happen at the same time).